sahi/model.py

```diff
@@ -1063,3 +1063,106 @@ def _create_object_prediction_list_from_original_predictions(
             object_prediction_list_per_image.append(object_prediction_list)
 
         self._object_prediction_list_per_image = object_prediction_list_per_image
+
+
+@check_requirements(["tensorflow", "tensorflow_hub"])
+class TensorflowHubDetectionModel(DetectionModel):
+    def load_model(self):
+        try:
+            import tensorflow_hub as hub
+
+        except ImportError:
+            raise ImportError(
+                "Please install tensorflow and tensorflow_hub via `pip install tensorflow' and 'pip install tensorflow_hub`"
+            )
+
+        self.model = hub.load(self.model_path)
+
+    def perform_inference(self, image: np.ndarray):
+        if self.image_size is not None:
+            from sahi.utils.tfhub import get_image, resize
+
+            img = get_image(image)
+            img = resize(img, self.image_size)
+
+            prediction_result = self.model(img)
+
+        else:
+            from sahi.utils.tfhub import get_image
+
+            img = get_image(image)
+
+            prediction_result = self.model(img)
+
+        self._original_predictions = prediction_result
+        self.img = get_image(image)
```

Comment:
goruntunun tamamini kaydetmek saglikli degil, sadece size bilgisini kaydedebilirsin

ornek kullanim icin Huggingface Detector implementasyonuna bakabilirsin: https://github.com/obss/sahi/blob/57f70490364bff34fc74cea9ef5acfa7466556de/sahi/model.py#L772

Reply:
Birebir aynısını yapmadım. Şu şekilde kodlamak verimsiz olursa huggingface şeklinde güncelleyebilirim.

Reply:
```
self.image_height, self.image_width = image.shape[0], image.shape[1]
```

Reply:
tamamdir simdilik boyle yapalim `# TODO: add support for multiple image prediction` diye ekleyebilirsen ilerisi icin cok iyi olur